### PR TITLE
Change rating values to 1,5,10

### PR DESCRIPTION
**Pull request by shubhamtewari**
note: 
- Also tested the new code style workflow and modified the checkstyle file accordingly.
- Will add comments to CM-80.

### CODE REVIEW

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java [hunk 1]
```diff
@@ -31,41 +31,42 @@
...
     public @interface Rating {
     }
 
-    @Rating
-    public int getRatingLevel() {
-        return ratingLevel;
-    }
+    public static final int[] FEEDBACK_RATING_VALUES = {RATING_POOR, RATING_AVERAGE, RATING_GOOD};
+
+    //1, 2 and 3 (POOR, AVERAGE and GOOD)
```
Comment (devashishmamgain) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java]: 1, 5 and 10

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java [hunk 2]
```diff
@@ -111,18 +112,18 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
...
+        feedbackRatingGroup.createViewForRatingLevel(view, 2, R.id.idButtonAverage, R.id.idTextAverage);
+        feedbackRatingGroup.createViewForRatingLevel(view, 3, R.id.idButtonGood, R.id.idTextGood);
 
         buttonSubmitFeedback.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                if (ratingLevel == 0) {
-                    setRatingLevel(RATING_AVERAGE);
+                if (ratingIndex == 0) {
+                    setRatingIndex(2);//average by default
```
Comment (devashishmamgain) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java]: use constant

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java (hunk 1 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java]: comment no longer required

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java (hunk 2 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java]: no longer required

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java (hunk 1 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java]: added intdef

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java (hunk 2 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java]: added type checking

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java [hunk 3]
```diff
@@ -31,41 +31,44 @@
...
+    private FeedbackRatingGroup feedbackRatingGroup;
 
-    FeedbackFragmentListener feedbackFragmentListener;
+    private FeedbackFragmentListener feedbackFragmentListener;
 
-    public static final int RATING_POOR = 1;
-    public static final int RATING_AVERAGE = 2;
-    public static final int RATING_GOOD = 3;
+    public static final int RATING_POOR = 0;
+    public static final int RATING_POOR_VALUE = 1;
```
Comment (devashishmamgain) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java]: do we need both? can we just keep 1, 5 and 10 constants?

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java [hunk 4]
```diff
@@ -4045,9 +4045,11 @@ public boolean onItemClick(int position, MenuItem item) {
                     attachReplyCancelLayout.setVisibility(VISIBLE);
                     replayRelativeLayout.setVisibility(VISIBLE);
                 } catch (Exception e) {
-
+                    e.printStackTrace();
```
Comment (devashishmamgain) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java]: @shubhamtewari @reytum do we use  e.printStackTrace(); at other places in the sdk too? or we use only log statement

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java (hunk 3 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java]: we need the indexes identify the level of ratings and to index the drawable arrays. the values of the ratings are seperate from their levels. Eg. rating of index one can have a value of 3 and index 2 can have a value of 7 etc

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java (hunk 4 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java]: on searching for printStackTrace, I found 100+ occurences. They were usually inside the catch block. Log is generally used on other cases.

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java (hunk 3 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java]: used list, removed constant

**Pull request merged**